dcodeKi-app/dcodeKi.py

```diff
@@ -141,15 +144,16 @@ def copy(self):
 		except:
 			self.ids.string.text = "Nothing to copy!"
 
-
+class SymbolCipherPage(Screen):
```

Comment:
name of the class in python file and kivy file doesn't match 
make it "SymbolCipherScreen"

while adding the widget use the same